syft/frameworks/torch/federated/dataloader.py

```diff
@@ -110,6 +110,50 @@ def stop(self):
         raise StopIteration
 
 
+class _DataLoaderOneWorkerIter(object):
+    r"""Iterates once over the worker's dataset, as specified by its sampler"""
```

Comment:
does this need to be a raw string literal?

Reply:
No, I don't think so. I will remove the r-prefix from all the docstrings in the file.